config/README.md

```diff
@@ -0,0 +1,125 @@
+# Rustic Configuration Specification
+
+Rustic is a backup tool that allows users to define their backup options using a TOML configuration file. The configuration file consists of various sections and attributes that control the behavior of Rustic for different commands and sources.
+
+This specification covers all the available sections and attributes in the Rustic configuration file. Users can customize their backup behavior by modifying these attributes according to their needs.
+
+## Merge Precedence
+
+The merge precedence for values is:
+
+    Commandline Arguments >> Environment Variables >> Configuration File
+
+Values parsed from the `configuration file` can be overwritten by `environment variables`, which can be overwritten by `commandline arguments` options. Therefore `commandline arguments` have the highest precedence.
+
+**NOTE**: There are the following restrictions:
+
+- You can overwrite values, but for most values, you cannot "unset" them on a higher priority level.
+
+- For some integer values, you cannot even overwrite with the value `0`, e.g. `keep-weekly = 5` in the `[forget]` section of the config file cannot be overwritten by `--keep-weekly 5`.
```

Comment:
should be: "cannot be overwritten by --- = 0"

Reply:
e.g. `keep-weekly = 5` in the `[forget]` section of the config file cannot be overwritten by `--keep-weekly 0`.